build/build.xml

```diff
@@ -20,8 +37,9 @@
     <echo message="JAVA   : ${java.version}" level="info" />
 
     <path id="compilepath">
+      <path refid="cp" />
       <fileset dir="${lib.dir}">
-        <include name="**/*.jar" />
+        <include name="nc.jar" />
```

Comment:
Bei Gelegenheit könnte man `nc.jar` in `nc-3.0.0.jar` umbenennen und eine reine distribution jar ohne source code daraus machen.

Reply:
Mein Vorschlag wäre genau die [eine Datei](https://github.com/jonelo/n16n-desktop/blob/master/src/net/numericalchameleon/util/spokennumbers/GermanNumber.java) zu extrahieren und Numerical Chameleon ansonsten nicht zu nutzen. Müsste lizenztechnisch unproblematisch sein, da wir ebenfalls unter GPL lizensieren und den Source öffentlich anbieten. Etwaige Anpassungen können ungehindert zum ursprünglichen Autor fließen.

Reply:
Danke, dass du schon mal gecheckt hast, was wir von der lib überhaupt benutzen.
Ich bin auch dafür, diese Datei zu extrahieren. Dann wäre die Dependency raus. Kannst du dafür eine PR erstellen?

Reply:
Sobal der durch ist, würde ich  das machen.